editor/src/components/canvas/commands/commands.ts

```diff
@@ -163,22 +163,33 @@ export function foldAndApplyCommandsInner(
   let workingCommandDescriptions: Array<CommandDescription> = []
 
   const runCommand = (command: CanvasCommand, shouldAccumulatePatches: boolean) => {
-    if (transient === 'transient' || command.transient === 'permanent') {
+    const shouldRunCommand: boolean = (() => {
```

Comment:
I think this is clearer just making a `let` variable and assigning it in either branch of the if statement.

Reply:
sigh, I keep wishing that this (()=>{})() pattern somehow becomes better, but it never does